docs/playbook-reference/triggers/prometheus.rst

```diff
@@ -38,6 +40,10 @@ The following triggers are available for Prometheus alerts:
 
     .. pydantic-model:: robusta.integrations.prometheus.trigger.PrometheusAlertTrigger
 
+    The ``scope`` filtering mechanism works exactly as it does for sinks
+    (see :ref:`sink-scope-matching` for more information), but the elements that can
+    be matched are the ones available in Prometheus alert payload.
```

Comment:
here I would say it's only alert `labels` and `annotations`

Reply:
changed wording